src/lib/utils/compiler.h

```diff
@@ -156,32 +156,19 @@
 
 #endif
 
-/*
-* Define BOTAN_PARALLEL_SIMD_FOR
-*/
-#if !defined(BOTAN_PARALLEL_SIMD_FOR)
-
-   #if defined(BOTAN_BUILD_COMPILER_IS_GCC)
-      #define BOTAN_PARALLEL_SIMD_FOR _Pragma("GCC ivdep") for
-   #else
-      #define BOTAN_PARALLEL_SIMD_FOR for
-   #endif
-
-#endif
-
-#if defined(BOTAN_BUILD_COMPILER_IS_GCC)
-   #define BOTAN_DIAGNOSTIC_PUSH _Pragma("GCC diagnostic push")
-   #define BOTAN_DIAGNOSTIC_IGNORE_DEPRECATED_DECLARATIONS \
-      _Pragma("GCC diagnostic ignored \"-Wdeprecated-declarations\"")
-   #define BOTAN_DIAGNOSTIC_IGNORE_INHERITED_VIA_DOMINANCE
-   #define BOTAN_DIAGNOSTIC_POP _Pragma("GCC diagnostic pop")
-#elif defined(BOTAN_BUILD_COMPILER_IS_CLANG)
+#if defined(__clang__)
```

Comment:
Interesting catch! Maybe leave a note to future-us, why that isn't using `BOTAN_BUILD_COMPILER_IS_*`?

Reply:
Done